functions/index.js

```diff
@@ -381,3 +422,18 @@ exports.weeklyDigest = functions.pubsub.schedule('0 10 * * 2')
         return sendWeeklyDigestEmail(snapshot.docs);
       });
 });
+
+
+/**
+ * Every week, send a digest containing all of the submissions from the Tacoma collection from last week.
+ */
+exports.weeklyDigest = functions.pubsub.schedule('0 10 * * 2')
```

Comment:
Is it okay to overwrite the `weeklyDigest ` function here?

Reply:
Oops! I have updated the function name to weeklyDigestTacoma